client/src/modules/user/model/actions.js

```diff
@@ -19,6 +20,15 @@ const loginSuccess = data => ({
   payload: data
 });
 
+const fetchUserDetailsSuccess = data => ({
```

Comment:
Consider renaming the parameter from `data` to `payload` to indicate that is passed without any processing and to use shorthand syntax.